src/test/java/api/requests/ItemsLostRequiringActualCostsTests.java

```diff
@@ -0,0 +1,211 @@
+package api.requests;
+
+import static api.support.JsonCollectionAssistant.getRecordById;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTimeZone.UTC;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import org.folio.circulation.support.http.client.Response;
+import org.hamcrest.CoreMatchers;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import api.support.builders.ItemBuilder;
+import api.support.builders.LostItemFeePolicyBuilder;
+import api.support.http.IndividualResource;
+import api.support.http.ItemResource;
+import api.support.http.ResourceClient;
+import api.support.matchers.ItemMatchers;
+import api.support.spring.SpringApiTest;
+import io.vertx.core.json.JsonObject;
+
+public class ItemsLostRequiringActualCostsTests extends SpringApiTest {
+
+  public ItemsLostRequiringActualCostsTests() {
+    super(true, true);
+  }
+
+  @Test
+  public void isEmptyWhenNoLostItems() {
+    createClosedItem();
+    createOpenItem();
+
+    List<JsonObject> items = ResourceClient.forItemsLostRequiringActualCosts().getAll();
+
+    assertTrue(items.isEmpty());
+  }
+
+  @Test
+  public void hasManyItemsLost() {
```

Comment:
As this is the typical way the report will work, I think it makes sense for this to be the first test in the class.